extensions/dataspace-authority-spi/src/main/java/org/eclipse/dataspaceconnector/registration/authority/model/Participant.java

```diff
@@ -48,6 +53,11 @@ public ParticipantStatus getStatus() {
         return status;
     }
 
+    @Override
+    public Map<String, String> getTraceContext() {
+        return unmodifiableMap(traceContext);
```

Comment:
Not needed

Reply:
why it's not needed?

Reply:
Because it’s already made unmodifiable in setter

Reply:
We were looking at the StatefulEntity in the EDC to have the reference how the tracecontext is handled there: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/94563d2a7bc0e6a2ada4296f4ccb790adc8f173c/spi/common/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/entity/StatefulEntity.java
Maybe it should be also updated to keep the traceContext immutable after created in Builder

Reply:
Since we are decorating it in the builder, we can save the second decorator here